libs/ai-endpoints/langchain_nvidia_ai_endpoints/chat_models.py

```diff
@@ -220,14 +220,14 @@ def __init__(self, **kwargs: Any):
         super().__init__(**kwargs)
         self._client = _NVIDIAClient(
             base_url=self.base_url,
-            model=self.model,
-            default_model=self._default_model,
+            model_name=self.model,
+            default_hosted_model_name=self._default_model,
```

Comment:
I guess this worked because test case was not specific in this case
Can you add a test case for this?

```

def test_default_hosted(public_class: type) -> None:
    """
    Test that a model in the model table will be accepted.
    """
    # check if default model is getting set
    with pytest.warns(UserWarning):
        x = public_class()
        assert x.model == x._default_model_name
```

Reply:
i'm adding a test for default being hosted and the default being set to _default_model_name